src/components/tasks/TaskWrapper.js

```diff
@@ -4,15 +4,15 @@ import {useParams} from "react-router-dom";
 import {codeCheckerLink} from "../../utils/apis/APILinks";
 import {useAbly} from "../../utils/hooks/useAbly";
 
-export const TaskWrapper = ({ id: propId, toSolveTask = true, submitLeftOffset = 0 }) => {
+export const TaskWrapper = ({ id: propId, toSolveTask = true, submitLeftOffset = 0, fetchTasks: propFetchTasks }) => {
 
     let { taskStageUUID } = useParams()
 
     if(!taskStageUUID) taskStageUUID = propId
 
     const { pub, sub, clientId } = useAbly(`codeChanged/${taskStageUUID}`)
 
-    const fetchTasks = toSolveTask ? () => tasksApi.getTasks(taskStageUUID || NaN) : () => new Promise(_ => {})
+    const fetchTasks = !!propFetchTasks ? propFetchTasks : toSolveTask ? () => tasksApi.getTasks(taskStageUUID || NaN) : () => new Promise(_ => {})
```

Comment:
Czy tu jest podwójnym ternariusz? Jeśli tak to dodałbym nawiasy bo nie umiem sobie wyobrazić jak się to ewaluuje

Reply:
No po kolei się ewaluuje. Pierwszy warunek, jeśli nie, to drugi warunek XDD

Reply:
Nadal wydaje mi się żeby to jakoś zmienić . Albo przynajmniej rozdzielić do dwóch linijek na osobne ternariusze, ale może przesadzam XDD

Reply:
Nie jest to złe, ale też bym to jakoś inaczej sformatował